apps/commercetools/src/api/products.ts

```diff
@@ -5,14 +5,12 @@ import { createClient } from './client';
 
 const MAX_LIMIT = 500;
 
-function productTransformer({ projectKey, locale }: ConfigurationParameters) {
+function productTransformer({ projectKey, locale, apiEndpoint }: ConfigurationParameters) {
+  const baseUrl = apiEndpoint?.split('api.')[1];
   return (item: ProductProjection): Product => {
     const id = item.id ?? '';
     const externalLink =
-      (projectKey &&
-        id &&
-        `https://mc.europe-west1.gcp.commercetools.com/${projectKey}/products/${id}/general`) ||
-      '';
+      (projectKey && id && `https://mc.${baseUrl}/${projectKey}/products/${id}/general`) || '';
```

Comment:
Are we sure that the merchant center and API URL always exactly match in that way? Is there docs on how that URL is created?

Reply:
I don't believe so. We could implement a new field in the config for the merchant center url, and fall back to the European address.